### PR TITLE
Make /play/tutorial URL work

### PR DESCRIPTION
**Pull request by AlexNisnevich**

### CODE REVIEW

File: src/common/containers/GameArea.js [hunk 1]
```diff
@@ -234,7 +238,11 @@ export class GameArea extends Component {
     super(props);
 
     if (!props.started) {
-      this.props.history.push('/play');
+      if (props.history.location.pathname.includes('/tutorial')) {
```
Comment (john-patterson) [inline src/common/containers/GameArea.js]: The `/tutorial` and `/play` string constants should probably be extracted from this location.

Comment (john-patterson) [reply src/common/containers/GameArea.js]: `props.history.location.pathname.includes` is also a bit wordy. Maybe a wrapper of `history` is in order, could be more concise as `props.history.hasPart('/tutorial')` or something similar.

File: src/common/containers/GameArea.js [hunk 2]
```diff
@@ -234,7 +238,11 @@ export class GameArea extends Component {
...
+        props.onStartTutorial();
+      } else {
+        props.history.push('/play');
```
Comment (john-patterson) [inline src/common/containers/GameArea.js]: Where does this play URL part get consumed?

Comment (AlexNisnevich) [reply src/common/containers/GameArea.js]: It goes to `containers/Play.js`, which renders the main lobby.

File: src/common/containers/GameArea.js (hunk 1 above)
Comment (AlexNisnevich) [reply src/common/containers/GameArea.js]: @john-patterson How about now? I made things a bit more explicit.

File: src/common/containers/GameArea.js [hunk 3]
```diff
@@ -237,11 +240,17 @@ export class GameArea extends Component {
...
+    // If the game hasn't started yet, that means that the player got here
+    // by messing with the URL (rather than by clicking a button in the lobby).
+    // If the URL is '/play/tutorial', just start the tutorial.
+    // Otherwise, return to the lobby because we can't do anything else.
+    if (!started) {
+      if (location.pathname.startsWith(Play.urlForGameMode('tutorial'))) {
+        onStartTutorial();
       } else {
-        props.history.push('/play');
+        history.push(Play.urlForGameMode(null));
```
Comment (john-patterson) [inline src/common/containers/GameArea.js]: I don't like the passing `null` here as a convention. Maybe two different methods. `urlForTutorialMode` and `urlForPlayMode`?

Comment (AlexNisnevich) [reply src/common/containers/GameArea.js]: @john-patterson How about now?

**Comment by john-patterson** [comment]
Lgtm!
